app/src/main/res/values-de-rDE/strings.xml

```diff
@@ -0,0 +1,185 @@
+<resources>
+    <string name="app_name" translatable="false">PeakAR</string>
```

Comment:
Cirrus complains that this is translated in German. Since it is set to false it shouldn't

Reply:
Oh yes, I just copied the default strings.xml (english version) and didn't remove the translatable=false strings